hack/mk-leader-cluster.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
```

Comment:
Can we call it manager rather than leader? To align the vocab with KEP, see the naming discussion in the KEP PR: https://github.com/kubernetes-sigs/kueue/pull/1380#discussion_r1409336565.

Reply:
done